x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_anomalous_user_name.json

```diff
@@ -1,66 +1,27 @@
 {
   "description": "Security: Windows - Rare and unusual users that are not normally active may indicate unauthorized changes or activity by an unauthorized user which may be credentialed access or lateral movement.",
-  "groups": [
-    "endpoint",
-    "event-log",
-    "process",
-    "security",
-    "sysmon",
-    "windows",
-    "winlogbeat"
-  ],
+  "groups": ["security", "windows"],
   "analysis_config": {
     "bucket_span": "15m",
     "detectors": [
       {
-        "detector_description": "Detects rare user.name values.",
+        "detector_description": "Detects rare user names.",
```

Comment:
There is a mix of `user name` and `username` throughout the security jobs. Should we be using one or the other? A lot of the other jobs use `Detects rare usernames.`